witness/src/Dscp/Resource/Network.hs

```diff
@@ -123,9 +126,15 @@ instance WithNetLogging => AllocResource NetServParams NetServResources where
         buildComponentR "netcli" allocate release
       where
         allocate = do
+            nsPeers' <- liftIO $ Set.fromList <$> mapM mkZTNodeId (Set.toList nsPeers)
+            nsOurAddress' <- liftIO $ mkZTNodeId nsOurAddress
+            nsInternalAddress' <-
+                liftIO $ maybe (pure Nothing) (fmap Just . mkZTNodeId) nsInternalAddress
+
             global <- ztGlobalEnv (unNetLogging netLogging)
-            cli <- createNetCliEnv global nsPeers
-            serv <- createNetServEnv global nsOurAddress nsInternalAddress
+            cli <- createNetCliEnv global nsPeers'
+            serv <- createNetServEnv global nsOurAddress' nsInternalAddress'
```

Comment:
Also, lets not use names with `'`. Its easy to forgot `'` and get yourself incorrect behaviour or bottom and an hour of wtf-debug.

Lets call them `new-` and `old-` instead. Or `src-`/`tmp-`/`dst-`.

Reply:
ok